tests/lora/utils.py

```diff
@@ -414,11 +421,14 @@ def test_simple_inference_with_text_lora_save_load(self):
             pipe.set_progress_bar_config(disable=None)
             _, _, inputs = self.get_dummy_inputs(with_generator=False)
 
-            output_no_lora = pipe(**inputs, generator=torch.manual_seed(0)).images
+            output_no_lora = pipe(**inputs, generator=torch.manual_seed(0))[0]
             self.assertTrue(output_no_lora.shape == self.output_shape)
 
-            pipe.text_encoder.add_adapter(text_lora_config)
-            self.assertTrue(check_if_lora_correctly_set(pipe.text_encoder), "Lora not correctly set in text encoder")
+            if self.test_text_encoder_lora:
```

Comment:
I understand that we do not currently have text encoder lora for cogvideox - but this test should not fail for cogvideox if we enabled it, no?

Reply:
actually, can we use this flag instead
`if "text_encoder" in self.pipeline_class._lora_loadable_modules:`